src/App.tsx

```diff
@@ -22,20 +22,20 @@ const App = () => {
 							<Route exact path="/">
 								<Home/>
 							</Route>
-							<Route path="/login">
+							<Route path="/temp-login">
```

Comment:
any reason to use temp

Reply:
yeah because our current auth routes (`/login`, `/signup`) are conflicting with these.

Reply:
ok